B827EBFFFEBBF413.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEBBF413", 
+		"servers": [ 
+			{ 
+				"server_address": "eu.thethings.network", 
```

Comment:
The router address is incorrect, please compare with the others or check the docs; update it and then I will merge. Cheers!